src/async_client.rs

```diff
@@ -20,6 +20,17 @@ use crate::{
     ServiceResponse,
 };
 
+/// Timeout for `UA_Client_run_iterate()`.
+///
+/// This is the maximum amount of time that `UA_Client_run_iterate()` will block for. It is relevant
+/// primarily when canceling the background task, i.e. when we need to interrupt the loop and cancel
+/// before the next invocation of `UA_Client_run_iterate()`.
+///
+/// Since this is also the timeout we must block for when dropping the client without `disconnect()`
+/// first, the value should not be too large. On the other hand, it should not be too small to avoid
+/// repeatedly calling `poll()`/`select()` inside open62541's event loop implementation.
+const ITERATION_TIMEOUT: Duration = Duration::from_millis(500);
```

Comment:
```suggestion
const RUN_ITERATE_TIMEOUT: Duration = Duration::from_millis(500);
```

Reply:
How about reducing this value to 100 ms (or at least 200 ms)? Polling 10x per second is not much and reduces the latency on disconnect and especially when dropping the client.

Hard-coded values may not suit everyone. But I agree that this parameter doesn't need to be customized with the new background thread until such a requirement emerges.